client/imports/account/account.html

```diff
@@ -0,0 +1,54 @@
+<ul id = "toolbar">
+  <li><a class="active" href="./home">Home</a></li>
+  <li><a href="./account">Account</a></li>
+  <li><a href="./coursepage">Courses</a></li>
+</ul>
+
+<style>
```

Comment:
I am not sure if this is from your or Jessica's code, but you should not have a static toolbar on every page.  It should be a component which is being imported.  Instead, you should be creating a Layout Template, so that the toolbar can be edited once, and not on every single page of the site if a change needs to be made.  This was covered in the tutorial here (http://www.angular-meteor.com/tutorials/socially/angular2/routing-and-multiple-views#multipleviewsandlayouttemplate).